testsuite/MDAnalysisTests/analysis/test_hbond_analysis.py

```diff
@@ -0,0 +1,72 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+import numpy as np
+import MDAnalysis
+from MDAnalysis.analysis.hbond_analysis import HydrogenBondAnalysis
+
+import pytest
+from numpy.testing import assert_allclose
+from MDAnalysisTests.datafiles import waterPSF, waterDCD
+
+
+class TestHydrogenBondAnalysisTIP3P(object):
+
+    @staticmethod
+    @pytest.fixture()
+    def universe():
+        return MDAnalysis.Universe(waterPSF, waterDCD)
+
+    kwargs = {
+        'donors_sel': 'name OH2',
+        'hydrogens_sel': 'name H1 H2',
+        'acceptors_sel': 'name OH2',
+        'd_h_cutoff': 1.2,
+        'd_a_cutoff': 3.0,
+        'd_h_a_angle_cutoff': 120.0
+    }
+
+    @pytest.fixture()
+    def h(self, universe):
+        h = MDAnalysis.analysis.hbond_analysis.HydrogenBondAnalysis(universe, **self.kwargs)
+        h.run()
+        return h
+
+    def test_hbond_analysis(self, h):
+
+        h = h
```

Comment:
That's weird, does pytest make you do this? You should be able to omit this line.

Reply:
It is weird. I've deleted it.